package.json

```diff
@@ -602,6 +653,29 @@
           "type": "boolean",
           "default": false,
           "description": "Specifies whether notifications should be shown if OmniSharp encounters warnings or errors loading a project. Note that these warnings/errors are always emitted to the OmniSharp log"
+        },
+        "omnisharp.preview": {
+          "type": "boolean",
+          "default": false,
+          "description": "Specifies whether to enable OmniSharp preview features."
+        },
+        "razor.languageServer.path": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "default": null,
+          "description": "Overrides the path to the Razor LanguageServer. If specified, this should be the absolute path to the language server directory."
+        },
+        "razor.languageServer.debug": {
+          "type": "boolean",
+          "default": false,
+          "description": "Specifies whether to wait for debug attach when launching the language server."
+        },
+        "razor.languageServer.trace": {
+          "type": "string",
+          "default": "Messages",
+          "description": "Specifies whether to output all messages [Verbose], some messages [Messages] or not at all [Off]."
```

Comment:
Are these the only possible inputs the user must specify. If so, we can use an enum type here, similar to https://github.com/OmniSharp/omnisharp-vscode/blob/master/package.json#L528

Reply:
Good tip - thanks. Done.